e2e/benchmarks/model_config.js

```diff
@@ -196,12 +196,26 @@ const benchmarks = {
   },
   'posenet': {
     type: 'GraphModel',
-    supportedInput: [257, 512, 1024],
-    load: async (inputResolution = 257) => {
-      const mobileNetConfig = {
-        inputResolution: inputResolution,
-      };
-      const model = await posenet.load(mobileNetConfig);
+    inputSizes: [128, 257, 512, 1024],
+    architectures: ['MobileNetV1', 'ResNet50'],
+    load: async (inputResolution = 257, modelArchitecture = 'MobileNetV1') => {
+      let config = null;
+      if (modelArchitecture === 'MobileNetV1') {
+        config = {
+          architecture: modelArchitecture,
+          outputStride: 16,
+          multiplier: 0.75,
+          inputResolution: inputResolution,
+        };
+      } else if (modelArchitecture === 'ResNet50') {
+        config = {
+          architecture: modelArchitecture,
+          outputStride: 32,
+          quantBytes: 2,
+          inputResolution: inputResolution,
```

Comment:
Why can't keep same `config` like below?
```
const config = {
          architecture: modelArchitecture,
          inputResolution: inputResolution,
}

Reply:
This is from tfjs-model

Reply:
Thanks for the clarification.